### DISCUSSION TITLE
Ender 3 S1 Pro - supported? touch screen coming?

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [mriscoc/Ender3V2S1#813] [Q&A]
Hi,

I'm a newbie, so hope I dont ask a too stupid question :)

Is my printer, the Ender 3 S1 Pro supported by this firmware? And since its mentioned, the touchscreen is not supported or?
Will it be supported sooner or later?

Creality made their touchscreen firmware open source some weeks ago?

Thank you!

**Comment by jessedmatlock** [comment]
+1 
I see your requirements [https://github.com/mriscoc/Ender3V2S1/wiki#requirements](url) only call for color screens with the encoder knob, not touch screens.. will these touch screen machines be included in the future update??
